plans/cdh2587.md

```diff
@@ -0,0 +1,16 @@
+## Status
+
+I have my development environment up and feel comfortable making small changes, but would like to understand more of the source code before making substantial changes.
+
+## Goal
+I am going to implement the following use cases:
+
+* As a selendroid developer, I will investigate issues related to having multiple or no Android Virtual Devices installed when running tests without an emulator already up and running.
```

Comment:
I think you need to rephrase this.  The selendroid developer is the end-user that you intend to benefit, but what are the benefits they will see?

For example:  As a selendroid developer with multiple Android Virtual Devices installed, I should be able to run tests and have them pass without already having an emulator up and running.